<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Path</class>
 <widget class="QDialog" name="Path">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Директория</string>
  </property>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: rgb(67, 98, 92);
	font: 15pt &quot;Segoe UI Variable Text&quot;;
    padding: 10px;
}
QPushButton:hover {
    background-color: rgb(54, 77, 70); 
}</string>
   </property>
   <property name="text">
    <string>Cохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 5px;
	background-color: rgb(67, 98, 92);
	font: 15pt &quot;Segoe UI Variable Text&quot;;
    padding: 10px;
}
QPushButton:hover {
    background-color: rgb(54, 77, 70); 
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="Text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Выберите директорию к вашим файлам:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InputPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ваш путь</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
